ui/js/reactrevisions.jsx

```diff
@@ -18,7 +18,9 @@ const RevisionItem = (props) => {
     let email, name, userTokens, escapedComment, escapedCommentHTML, initialsHTML, tags;
 
     userTokens = props.revision.author.split(/[<>]+/);
-    name = userTokens[0].trim();
+    name = userTokens[0].trim().replace(/\w\S*/g,
+                                        txt => txt.charAt(0).toUpperCase() + txt.substr(1));
```

Comment:
If you want this 100+ chars long let me know and I'll unify the line. I left it on two lines for now.